cdi/src/main/java/org/hibernate/validator/cdi/ValidationExtension.java

```diff
@@ -51,8 +51,8 @@
 import org.hibernate.validator.cdi.internal.ValidationProviderHelper;
 import org.hibernate.validator.cdi.internal.ValidatorBean;
 import org.hibernate.validator.cdi.internal.ValidatorFactoryBean;
-import org.hibernate.validator.cdi.internal.interceptor.ValidationEnabledAnnotatedType;
-import org.hibernate.validator.cdi.internal.interceptor.ValidationInterceptor;
+import org.hibernate.validator.cdi.interceptor.spi.internal.ValidationEnabledAnnotatedType;
```

Comment:
`.spi.internal`, really? :thinking: 

Are you trying to confuse automatic tooling? :)